onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -25,6 +25,11 @@ class Logger;
 
 namespace webnn {
 
+enum WnnDeviceType {
```

Comment:
[naming suggestion] Recommend `WebnnDeviceType`, just because it's more readable and matches the namespace and provider folder. I haven't heard WNN used as an acronym before (I guessed what it meant, but others might not so readily).

Reply:
Better as `enum class`? Except when used as indices or bitmasks, I generally opt for `enum class`, just because the enumerants to not leak into the outer scope.

Reply:
Thanks! Done, PTAL again.